core/src/avm2/globals/flash/text/text_field.rs

```diff
@@ -32,6 +32,24 @@ pub fn init<'gc>(
     Ok(Value::Undefined)
 }
 
+pub fn get_always_show_selection<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    _this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    avm2_stub_getter!(activation, "flash.text.TextField", "alwaysShowSelection");
+    Ok(Value::Bool(false))
```

Comment:
This can also be accomplished using `false.into()` rather than `Value::Bool(false)`.